src/sentry/snuba/outcomes.py

```diff
@@ -263,40 +274,58 @@ def __init__(
             query_groupby.update(groupby.get_snuba_groupby())
         self.query_groupby = list(query_groupby)
 
-        self.conditions, self.filter_keys = get_filter(query, params)
+        self.group_by = []
+        for key in self.query_groupby:
+            self.group_by.append(Column(key))
+
+        conditions, filter_keys = get_filter(query, params)
+
+        for org in filter_keys.get("org_id", []):
```

Comment:
Do we ever query for more than one org? That seems like a scenario that should cause a failure, as should zero organizations.

Reply:
I didn't have that at first but I think I was failing something on test_outcomes, lemme remove it and see if it passes

Reply:
If i remove it, it fails 4 of the test_outcomes tests it seems

Reply:
@maxiuyuan there should only ever be one org passed in, how are the tests failing? what are you replacing it with?

Reply:
I dont believe i touched the ones that failed, when it makes a query it does the following in test_correct_times_seen_aggregate:
![image](https://user-images.githubusercontent.com/22259802/127529287-ffb04e32-5821-4bcc-9801-14e03a011ba0.png)
which doesn't pass in the org id like it does in:
![image](https://user-images.githubusercontent.com/22259802/127529337-4c069f68-fe4e-4735-9a3f-92a050a7ee0c.png)
i can remove the default value of [] if theres no org stats if i change the test to include an org id i think

Reply:
ah okay, you can go ahead and add an organization_id to the test params, thanks!

Reply:
alright for sure!